material-overrides/main.html

```diff
@@ -58,4 +66,107 @@
         : (link.href = "{{ 'assets/images/favicon-light-mode.png' | url }}");
     }
   </script>
+
+  <script>
```

Comment:
This script differs slightly from the one in Wormhole, but the goal is the same. I’d prefer they be more closely aligned so we build familiarity with a single approach.